CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to Softeq .NET Guidelines
+The repository is maintained using [Scaled Trunk-Based Development practices](https://trunkbaseddevelopment.com/).
```

Comment:
Not sure if that makes sense, but at least to me if you add at the end of the sentence aka GitHub Flow. Because probably most of us are familiar with that particular name due to its popularity. @oleg-zaitsev @smolyakoff what do you think?

Reply:
@eabliazimov, TBD and GitHub Flow are not the same. The main difference (why I've chosen TBD) is that TBD explicitly introduces version branches (https://trunkbaseddevelopment.com/branch-for-release/) while version tags are more common for GitHub Flow. Having separate version branches will allow the teams to apply hotfixes without migration to later guidelines versions.

Reply:
@oleg-zaitsev oh, got it. Thank you for the explanation, I missed that point than and it is confused me. So the production deployment going from the release branch instead of a Trunk in this case to prevent blockers for the on-going development.